scripts/aem.js

```diff
@@ -453,7 +449,9 @@ function decorateSections(main) {
         const wrapper = document.createElement('div');
         wrappers.push(wrapper);
         defaultContent = e.tagName !== 'DIV';
-        if (defaultContent) wrapper.classList.add('default-content-wrapper');
+        if (defaultContent) {
```

Comment:
why not put direct condition in if, if this var is not used anywhere else?
```
if (e.tagName !== 'DIV')
```

also remove declaration of it

Reply:
I put it back the way it was. 🙂
`aem.js` comes directly from the boilerplate. We try to the keep changes here to a minimum. If something is absolutely needed we do it, but otherwise we try to keep it as is.
even the button code change looks like I might be putting back.